src/uct/base/uct_iface.c

```diff
@@ -860,21 +860,20 @@ ucs_status_t uct_ep_keepalive_check(uct_ep_h ep, uct_keepalive_info_t **ka_p,
                                     pid_t pid, unsigned flags,
                                     uct_completion_t *comp)
 {
-    struct timespec create_time;
+    ucs_status_t status = UCS_OK;
+    unsigned long start_time;
     uct_keepalive_info_t *ka;
-    ucs_status_t status;
 
     UCT_EP_KEEPALIVE_CHECK_PARAM(flags, comp);
 
     if (*ka_p == NULL) {
-        status = uct_ep_keepalive_create(pid, ka_p);
+        status     = uct_ep_keepalive_create(pid, ka_p);
     } else {
-        ka     = *ka_p;
-        status = ucs_sys_get_file_time(ka->proc, UCS_SYS_FILE_TIME_CTIME,
-                                       &create_time);
-        if ((status != UCS_OK) ||
-            (ka->start_time.tv_sec != create_time.tv_sec) ||
-            (ka->start_time.tv_nsec != create_time.tv_nsec)) {
+        ka         = *ka_p;
+        start_time = ucs_sys_get_proc_create_time(ka->proc);
+        if (ka->start_time != start_time) {
+            ucs_diag("ka failed: %s start time %lu != %lu", ka->proc,
```

Comment:
print pid

Reply:
the pid will be printed as a path to stat file by `ka->proc`. Do you mean to print pid instead of the path or in addition?